spec/posix/1.0.md

```diff
@@ -28,3 +28,18 @@ Therefore, VT6 clients MUST use the following method to determine whether a VT6
 2. If the `TERM` environment variable is present and contains the string `vt6`, the VT6 client SHALL assume that a VT6 server is present, and operate in multiplexed mode.
 
 3. If the `VT6` environment variable is absent and the `TERM` environment variable does not contain the string `vt6`, the VT6 client MUST consider the VT6 server to be absent.
+
+## 3. Definitions for the `vt6/sig1` module
+
+VT6 signals are dispatched to clients by sending the corresponding POSIX signals to the client processes:
+
+- The `sig1.interrupt` message corresponds to the `SIGINT` signal.
+- The `sig1.quit` message corresponds to the `SIGQUIT` signal.
+- The `sig1.suspend` message corresponds to the `SIGSTOP` signal.
```

Comment:
Not sure if SIGSTOP is correct here, or if we should use SIGTSTP. See [this question of mine on StackExchange.](https://unix.stackexchange.com/q/410088/21622)

Reply:
Since platform-specific signals happen only *behind* the shell (which is the signal dispatcher), we don't need SIGTSTP, but can use SIGSTOP always.